doc/scripts/genrest/genrest.py

```diff
@@ -31,12 +32,23 @@ def print_items(items, outdir, indent):
             var = "CONFIG_%s" %item.get_name()
             if not var in done:
                 done.append(var)
-                f.write("   * - :option:`%s`\n" %var)
+
+                # Instead of writing the index document with symbols listed
+                # in the order discovered, save up the items in a dictionary
+                # (itemIndex) and then write out the index doc with options in
+                # alphabetic order, together with the "prompt" (short
+                # description) from the Kconfig file.
+
+                #f.write("   * - :option:`%s`\n" %var)
                 if len(item.get_prompts()) > 0:
                     p = item.get_prompts()[0]
                 else:
                     p = ""
-                f.write("     - %s\n" %p)
+                #f.write("     - %s\n" %p)
```

Comment:
good point.  I've cleaned that up now, and also updated comment to not talk about how it used to be, and rather how it is now.